python_modules/jupedsim/jupedsim/native/models.py

```diff
@@ -1,49 +1,32 @@
 # Copyright © 2012-2023 Forschungszentrum Jülich GmbH
 # SPDX-License-Identifier: LGPL-3.0-or-later
 
+from dataclasses import dataclass
+
 try:
     import py_jupedsim as py_jps
 except ImportError:
     from .. import py_jupedsim as py_jps
 
 
-class VelocityModelBuilder:
-    def __init__(
-        self, a_ped: float, d_ped: float, a_wall: float, d_wall: float
-    ) -> None:
-        self._obj = py_jps.VelocityModelBuilder(
-            a_ped=a_ped, d_ped=d_ped, a_wall=a_wall, d_wall=d_wall
-        )
-
-    def build(self):
-        return self._obj.build()
-
-
-class GCFMModelBuilder:
-    def __init__(
-        self,
-        nu_ped: float,
-        nu_wall: float,
-        dist_eff_ped: float,
-        dist_eff_wall: float,
-        intp_width_ped: float,
-        intp_width_wall: float,
-        maxf_ped: float,
-        maxf_wall: float,
-    ) -> None:
-        self._obj = py_jps.GCFMModelBuilder(
-            nu_ped=nu_ped,
-            nu_wall=nu_wall,
-            dist_eff_ped=dist_eff_ped,
-            dist_eff_wall=dist_eff_wall,
-            intp_width_ped=intp_width_ped,
-            intp_width_wall=intp_width_wall,
-            maxf_ped=maxf_ped,
-            maxf_wall=maxf_wall,
-        )
-
-    def build(self):
-        return self._obj.build()
+@dataclass(kw_only=True)
+class VelocityModelParameters:
+    a_ped: float = 8.0
+    d_ped: float = 0.1
+    a_wall: float = 5.0
+    d_wall: float = 0.02
+
+
+@dataclass(kw_only=True)
+class GeneralizedCentrifugalForceModelParameters:
+    nu_ped: float
+    nu_wall: float
+    dist_eff_ped: float
+    dist_eff_wall: float
+    intp_width_ped: float
+    intp_width_wall: float
+    maxf_ped: float
+    maxf_wall: float
 
 
 class GCFMModelAgentParameters:
```

Comment:
Every other occurrence of GCFM has been renamed to full name, except this part. We should be consistent in the naming here.